### PR TITLE
remove misleading raise_for_status

### PR DESCRIPTION
**Pull request by TAOXUY**
When the response has error, the error detail is not exposed because of raise_for_status, so remove it and show the detailed message.

For instance, the detailed error is UNAUTHENTICATED:Method doesn't allow unregistered callers (callers without established identity). Please use API Key or other form of API consumer identity to call this API.
but we only got requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url.

### CODE REVIEW

File: endpoints/getting-started/clients/google-jwt-client.py [hunk 1]
```diff
@@ -69,7 +69,6 @@ def make_jwt_request(signed_jwt, url='https://your-endpoint.com'):
     }
     response = requests.get(url, headers=headers)
 
-    response.raise_for_status()
```
Comment (busunkim96) [inline endpoints/getting-started/clients/google-jwt-client.py]: @DanSanche Does `response.text` need to be returned or is it alright to do the `print()` inside the function?

The `print()` could be moved into the function. 

```python
print(response.text)
response.raise_for_status()
# [END endpoints_jwt_request]
return response.text
```

Comment (DanSanche) [reply endpoints/getting-started/clients/google-jwt-client.py]: I'm forgetting some of the context around this sample, but I think printing should be fine. That code block looks good to me

Comment (busunkim96) [reply endpoints/getting-started/clients/google-jwt-client.py]: @TAOXUY would you mind making that change?

File: endpoints/getting-started/clients/google-jwt-client.py [hunk 2]
```diff
@@ -69,8 +69,7 @@ def make_jwt_request(signed_jwt, url='https://your-endpoint.com'):
...
-    return response.text
+    print(response.text)
 # [END endpoints_jwt_request]
```
Comment (busunkim96) [inline endpoints/getting-started/clients/google-jwt-client.py]: @TAOXUY Thank you! Could you also add a `response.raise_for_status()`after the `print()`?

**Comment by TAOXUY** [comment]
Please take a look

**Comment by leahecole** [comment]
@Daniel-Sanche can you take another look at this please?

**Comment by TAOXUY** [comment]
@busunkim96, please take a look.

File: endpoints/getting-started/clients/google-jwt-client.py [hunk 3]
```diff
@@ -68,9 +68,8 @@ def make_jwt_request(signed_jwt, url='https://your-endpoint.com'):
         'content-type': 'application/json'
     }
     response = requests.get(url, headers=headers)
+    print(response.status_code, response.content)
```
Comment (busunkim96) [inline endpoints/getting-started/clients/google-jwt-client.py:71]:
```suggestion
    print(response.status_code, response.content)
    response.raise_for_status()
```

Hi - could you re-add `raise_for_status()` so an exception is thrown if the request fails?

Comment (TAOXUY) [reply endpoints/getting-started/clients/google-jwt-client.py:71]: Done

**Comment by TAOXUY** [comment]
@busunkim96, please take a look.

**Comment by busunkim96** [comment]
Thank you for the contribution @TAOXUY!

**Pull request merged**